packages/calypso-e2e/src/lib/components/editor-settings-sidebar-component.ts

```diff
@@ -110,21 +196,21 @@ export class EditorSettingsSidebarComponent {
 	 * @param {string} param1.password Password for the post. Normally an optinal value, this
 	 * 	must be set if the `visibility` parameter is set to `Password`.
 	 */
-	async setVisibility(
+	async selectVisibility(
```

Comment:
This method used to handle all of the process involved in setting the privacy option for an article.

In keeping with the current ethos of the refactor, the scope of this method has been reduced to handle just the interaction with the visibility popover.